### PR TITLE
CUDA 11.8 and 1.16.2 -- higher build number for CUDA

### PR DESCRIPTION
**Pull request by hmaarrfk**
Closes #81 
Closes #79 

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -5,14 +5,6 @@ MACOSX_SDK_VERSION:        # [osx and x86_64]
...
   - ""
   - "-novec"
-cudnn:                                            # [linux64]
-  - "8"                                           # [linux64]
-cuda_compiler_version:                            # [linux64]
-  - None                                          # [linux64]
-docker_image:                                     # [linux64]
-  - quay.io/condaforge/linux-anvil-cos7-x86_64    # [linux64]
-cdt_name:                                         # [linux64]
-  - cos7                                          # [linux64]
```
Comment (hmaarrfk) [inline recipe/conda_build_config.yaml:15]: was it ok for me to have removed these? I'm making the whole build cos7 ....

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by jakirkham** [comment]
CUDA 11.8 was just added in PR ( https://github.com/conda-forge/onnxruntime-feedstock/pull/87 )

This appears to be done for 1.6.3

Does that address this issue or is there more work still needed?

**Pull request closed without merging**